docs/api.rst

```diff
@@ -11213,6 +11213,138 @@ Get location labels
    :statuscode 500: Internal Rotki error
 
 
+ Lido CSM staking
+ =================
+
+.. http:get:: /api/(version)/lido-csm/node-operators
+
+   Return the list of tracked Lido Community Staking Module (CSM) node operators.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      GET /api/1/lido-csm/node-operators HTTP/1.1
+      Host: localhost:5042
+      Accept: application/json, text/javascript
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+          "result": [
+              {
+                  "address": "0x0000000000000000000000000000000000000000",
```

Comment:
Can you use a random address here? Zero address seems wrong as an example for any valid one